### PR TITLE
1.1.0 - Added implicit Play Json serialization code and tests.

### PR DESCRIPTION
**Pull request by jeffmay**
@yolo-swaggerson @rcmurphy

Added play json package to scala-enum project.

### CODE REVIEW

**Comment by rcmurphy** [comment]
Nice! Lgtm

File: build.sbt [hunk 1]
```diff
@@ -1,40 +1,66 @@
...
-  "-Xlint",
-  "-Ywarn-dead-code",
-  "-encoding", "UTF-8"
+lazy val commonRootSettings = Seq(
+  version := "1.1.0",
+  scalaVersion := "2.11.6",
+  crossScalaVersions := Seq("2.11.6", "2.10.4"),
+  organization := "me.jeffmay",
+  organizationName := "Jeff May"
+) ++ bintraySettings ++ bintrayPublishSettings
```
Comment (ghost) [inline build.sbt:8]: Same note here on using `in ThisBuild` instead of baseSettings for all of these.

**Comment by ghost** [comment]
LGTM

**Comment by jeffmay** [comment]
@chrissalij-r thanks!

File: build.sbt (hunk 1 above)
Comment (jeffmay) [inline build.sbt:8]: I'll look up how that works for next time.

File: build.sbt [hunk 2]
```diff
@@ -1,40 +1,66 @@
...
+
+commonRootSettings
```
Comment (yolo-swaggerson) [inline build.sbt:10]: What's the effect of this line?  Isn't `commonRootSettings` referenced in the very next expression?

**Pull request merged**